gobconfig/import_/data/test/test_AUTOID_STATES_PARTIAL.csv

```diff
@@ -0,0 +1,4 @@
+identificatie,volgnummer,begin_geldigheid,eind_geldigheid,code,autoid
+10,1,,,idx1
+10,2,,,idx1
+10,3,,,idx1
```

Comment:
Shouldn't the auto id columns be empty for some of the records if the bug which was happening in production should be tested? This occured when the partial import contained both records which already had an autoid assigned and some records which didn't receive an autoid.

I think the following row could be added:

12,1,,C,

Which should receive an autoid of 123400013 if everything works as expected

Reply:
You were right! thanks, the e2e test now fails for the original code but succeeds on the new code